src/worker/startup.ts

```diff
@@ -127,7 +127,18 @@ export async function start() {
 	queues.discovery.process(5, commonMiddleware(discovery(app, queues), DISCOVERY_LOGGER_NAME));
 	queues.installation.process(
 		Number(CONCURRENT_WORKERS),
-		commonMiddleware(processInstallation(app, queues), INSTALLATION_LOGGER_NAME)
+		commonMiddleware(processInstallation(app, queues, () => {
+			return Promise.resolve({
+				schedule: async (jobData, delayMsecs) => {
+					// TBD: switch to SQS with a FF
+					if ((delayMsecs || 0) > 0) {
```

Comment:
this can just be `if(delayMsecs)`  Or a simpler way of doing it is not having the conditional at all `await queues.installation.add(jobData, _.pickBy({delay: delayMsecs}));`